### ISSUE TITLE
Implement and test Mapbox

### ISSUE THREAD

**Issue by Kpaubert** [IT2901-SINTEF01/frontend#96]
### Description
It can be helpful to have data represented with the help of map, and Mapbox looks promising, having free tiers that we can use. With the help of the MetAPI, we can use the geo-data to see how well it works. _If this looks promising, it can be turned into a real PR_. This is also labelled as a question, in hopes that others will chime in with their opinions.

### Subtasks
- [ ] Create necessary components
- [ ] Make use of MetAPI
- [ ] Test out to see if it looks promising

### Related sources
- [Mapbox docs](https://docs.mapbox.com/)

**Comment by blauks** [comment]
I think it would be cool, but I am a little bit unsure where it would be logical to implement it. Under the visualisation editor maybe? Or if you press an edit data button, the map would show up in some sort of box.

But it would be nice to be able to choose weather data from other cities than Trondheim.

**Comment by Kpaubert** [comment]
Not sure where to implement either, but the visualisation editor seems good. It could either be used to display data, or as you inferred, let the user click on a place on the map to select an area to fetch data from. There are also API's that give you the municipality names based on lon/lat-values so the user can validate their selection (but that might exist in MapBox already).

With all that said, I'm struggling with the implementation. I might be doing something wrong with regards to yarn, but I'm asking August for some help later today so we can get a prototype running.

**Comment by FredrikAugust** [comment]
Could you update the issue with your thoughts, reflections, and plans @Kpaubert?

**Comment by FredrikAugust** [comment]
E.g. what will the map be used for?

**Comment by FredrikAugust** [comment]
And can we use mapbox or do we have to use an industry-leading technology such as Atlas? \s

**Comment by Kpaubert** [comment]
I moved over to using Reactive-Atlas because it was easier to work with, and it gave the information we needed (except it would be nice to get the municipality name simply from the click, without sending another 3rd party request). I think integrating this with the MetAPI or similar could be nice, although it's probably easier to use a select/dropdown with search-functionality to select the correct municipality.

The map could probably be postponed until we find a better usage for it. An example could be a query that relates data to a geographical spot that does not fit with one of the already established chart-types (e.g. line, threshold, etc.).

I do think mapbox would be the best alternative as a map-chart solution, as it has tonnes of functionality and seems quite easy to learn, but the problems with implementing it made for a real time-waster.

In conclusion: reactive-atlas works nicely. I think mapbox could work better. The whole map-chart should be postponed, and we should prioritize more pressing tasks.

@FredrikAugust LGTY?

**Comment by FredrikAugust** [comment]
LGTM. Sorry for late reply.

**Comment by Kpaubert** [comment]
No problem :) Closing this for now.
